src/parquet/util/buffer.h

```diff
@@ -127,12 +130,14 @@ class ResizableBuffer : public MutableBuffer {
 // garbage-collected
 class OwnedMutableBuffer : public ResizableBuffer {
  public:
-  OwnedMutableBuffer();
+  explicit OwnedMutableBuffer(MemoryPool* pool = get_default_pool());
```

Comment:
You *could* forward-declare `class MemoryPool;` then handle getting the default pool in `buffer.cc`, but not a big deal either way